google-cloud-datastore/src/main/java/com/google/cloud/datastore/DatastoreImpl.java

```diff
@@ -173,6 +190,8 @@ public T call() throws DatastoreException {
 
   com.google.datastore.v1.RunQueryResponse runQuery(
       final com.google.datastore.v1.RunQueryRequest requestPb) {
+    Span span = traceUtil.startSpan(TraceUtil.SPAN_NAME_RUNQUERY);
+    Scope scope = traceUtil.getTracer().withSpan(span);
```

Comment:
You can use try with resources for the scope (unless the order of closing the span would be wrong?)

Reply:
Done